src/data-visualisation/accessibility/design-a11y.md

```diff
@@ -0,0 +1,107 @@
+---
```

Comment:
If you are going to have directories within the `data-visualisation` folder you will need to add an `index.md` to each sub-directory. This way when clicking into the "Data visualisation standards" section you'd see a screen filled with your articles.

Reply:
Implemented this